doc/ChangeLog.md

```diff
@@ -16,20 +17,29 @@ All notable changes to the project are documented in this file.
  - Upgrade Buildroot to 2024.02.10 (LTS)
  - Add the possibility to change the boot order for the system with a
    RPC and add boot order to operational datastore.
+ - Add documentation on management via SSH, Web (RESTCONF, Web
+   Console), and Console Port, issue #787
+ - Add documentation of DNS client use and configuration, issue #798
  - SSH Server is now configurable, issue #441
    SSH Server and NETCONF Server now uses the same SSH hostkey in factory-config
  - Support for GRE/GRETAP tunnels
  - Support for STP/RSTP on bridges
  - Support for VXLAN tunnels
- - Add documentation on management via SSH, Web (RESTCONF, Web
-   Console), and Console Port, issue #787
+ - Add operational support for reading DNS resolver info, issue #510
+ - Add support for a basic DHCPv4 server, multiple subnets with static
```

Comment:
```suggestion
 - Add DHCPv4 server support, multiple subnets with static
```

Your wording is way to humble! This is far beyond basic, IMHO :muscle:

Reply:
Thanks mate, OK I'll change to your wording. I have a slew of local updates to the ChangeLog anyway :sweat_smile: